base/db/R/pft.add.spp.R

```diff
@@ -0,0 +1,121 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2012 University of Illinois, NCSA.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the 
+# University of Illinois/NCSA Open Source License
+# which accompanies this distribution, and is available at
+# http://opensource.ncsa.illinois.edu/license.html
+#-------------------------------------------------------------------------------
+## M. Dietze
+##' adds a list of species to a pft based on USDA Plants acronyms
+##'
+##' This function is used to add PFT-Species pairs to the database table 'pfts_species'.  In the initial implementation the PFT has to be defined already and the species are added based on their USDA Symbol (genus/species acronym).  Multiple species can be added at once but only one PFT at a time.
+##' @title Associate species with a PFT.
+##' @param pft String name of the PFT in the database
+##' @param acronym Specie's Symbols.
+##' @param ID Specie's IDs in Bety.
```

Comment:
1. function documentation is not making it clear that one needs to provide either the acronym or the ID. It's also not making clear where a valid acronym comes from. You probably need to add a @details section
2. `Specie's` is not a word. Please correct to `Species`

Reply:
@DongchenZ please fix